packages/slate/test/changes/at-current-range/set-block/across-blocks.js

```diff
@@ -26,7 +26,7 @@ export const output = (
         <anchor />word
       </code>
       <code>
-        <focus />another
+        a<focus />nother
```

Comment:
I think maybe this calls for an additional test.

1. That checks that the cursor being expanded but ending at the start of the block does NOT result in the focus block being changed
2. Basically what you implemented here.

Reply:
I was not sure how to proceed here so I put that logic in its own commit in case we wanted to revert it, I'll add a new test for the new behaviour (meaning restoring the old test but with the new behaviour).